configure_me/README.md

```diff
@@ -92,24 +107,36 @@ extern crate configure_me;
 include_config!();
 
 fn main() {
-    // Don't worry, unwrap_or_exit() prints a nice message instead of ugly panic
-    let (server_config, _remaining_args) = Config::including_optional_config_files(&["/etc/my_awesome_server/server.conf"]).unwrap_or_exit();
+    // This will read configuration from "/etc/my_awesome_server/server.conf" file and
+    // the command-line arguments.
+    let (server_config, _remaining_args) = Config::including_optional_config_files(&["/etc/my_awesome_server/server.conf]").unwrap_or_exit();
 
     // Your code here
     // E.g.:
     let listener = std::net::TcpListener::bind((server_config.bind_addr, server_config.port)).expect("Failed to bind socket");
 }
 ```
+Configuration
+-------------
+Configuration of the code generation is set in the same toml file as the specification
+```toml
+[general]
+# Prefix for all env vars - enables all env vars by default if present
+build = "build.rs"
```

Comment:
Did git screw up the diff somehow? env var prefix is configured using `env_prefix`, not `build`